docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:21.7.3-slim
+
+USER root
```

Comment:
Unnecessary.

```suggestion
```

Reply:
yep, you're right, it's already root by default at the start